src/OpenTelemetry/OpenTelemetry.csproj

```diff
@@ -26,6 +26,13 @@
 
   <ItemGroup>
     <ProjectReference Include="..\OpenTelemetry.Api\OpenTelemetry.Api.csproj" />
+
+    <PackageReference Include="Microsoft.CodeAnalysis.PublicApiAnalyzers" Version="3.0.0" Condition=" $(OS) == 'Windows_NT'">
+      <PrivateAssets>All</PrivateAssets>
+    </PackageReference>
```

Comment:
Changes are good for me. I'd wait to merge this PR for 2-3 days so that we can remove Span based public API and only have activity based ones in SDK.

Reply:
Ok! should I add the publicApi for the other projects as well?

Reply:
Lets do this PR along with adding to other projects after beta release. As we'll be changing public API before that anyway, and want to proceed fast without worrying about this txt file.